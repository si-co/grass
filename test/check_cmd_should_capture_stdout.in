login Acidburn
pass CrashOverride
rm ..
rm removeMe
exit
