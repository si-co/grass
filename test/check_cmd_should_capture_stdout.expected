Error : invalid path
Error : no such file or directory
